icloudpd/base.py

```diff
@@ -390,7 +391,10 @@ def download_photo(counter, photo):
             created_date = photo.created
 
         try:
-            date_path = folder_structure.format(created_date)
+            if folder_structure == "none":
```

Comment:
Is there a way to make this search case insensitive?

Reply:
good point.